File: Challenge/FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("Spider");
            } else if (i % 3 == 0) {
                System.out.println("Rat");
            } else if (i % 5 == 0) {
                System.out.println("Ghost");
            } else {
                System.out.println(i);
            }
        }
    }
}

File: Challenge/Python/fizzbuzz.py

#Python program to print Fizz Buzz loop for 100 times i.e. range 
for fizzbuzz in range(100):  

    if fizzbuzz % 15 == 0:  
        print("Spider")                                          
        continue

    elif fizzbuzz % 3 == 0:      
        print("Rat")                                          
        continue

    elif fizzbuzz % 5 == 0:          
        print("Ghost")                                      
        continue
     
    print(fizzbuzz) # print numbers
